src/_data/plugins/eleventy-plugin-clean-urls.json

```diff
@@ -0,0 +1,5 @@
+{
+	"npm": "@inframanufaktur/eleventy-plugin-clean-urls",
+	"author": "ovlb",
```

Comment:
@olivernorred Is this your Twitter username?

Reply:
I guess you wanted to tag me? My Twitter is @_ovlb